moto/rds/models.py

```diff
@@ -573,93 +743,115 @@ class DBInstance(CloudFormationModel, RDSBaseModel):
     resource_type = "db"
 
     def __init__(
-        self, backend: "RDSBackend", db_instance_identifier: str, **kwargs: Any
```

Comment:
I don't know whether this was intentional, but I do love the fact that we're getting rid of `kwargs` here. Typing them all out is a bit verbose, but I prefer the readability of it.

Reply:
Definitely intentional.  I plan to do it for all the RDS models.